NEWS.md

```diff
@@ -8,6 +8,7 @@ role as `datalimits` in `violin` [#2137](https://github.com/JuliaPlots/Makie.jl/
 - Improve performance around Axis(3) limits [#2115](https://github.com/JuliaPlots/Makie.jl/pull/2115).
 - Cleanup stroke artifacts in scatter and text [#2096](https://github.com/JuliaPlots/Makie.jl/pull/2096).
 - compile time improvements [#2153](https://github.com/JuliaPlots/Makie.jl/pull/2153).
+- interpolate between values instead of interpolating between colors for WGLMakie + GLMakie [#2097](https://github.com/JuliaPlots/Makie.jl/pull/2097)
```

Comment:
Capitalization here and above :)
It's not that good to paste the PR titles as those are usually not even complete sentences.